### DISCUSSION TITLE
Changing the grey title bar color when using ExtendsContentIntoTitleBar

### DISCUSSION THREAD

**Discussion by Fabi** [dotnet/maui#6397] [Q&A]
Hey!

is there finally a way to change the grey title bar color when using ExtendsContentIntoTitleBar without creating your own title bar and menu buttons?

They documentation on these things is horrible.

**Comment by Fabi** [comment]
Ok, I found this WinUI sample https://github.com/nickrandolph/AppWindowSample and this actually let me set this on Windows through getting the AppWindow.

One note is, that you HAVE to set `app.ExtendsContentIntoTitleBar = false;` in the windows App.xaml.cs over the `windows.OnPlatformMessage((app, args) =>` code part. Then through the `windows.OnWindowCreated(xamlWindow =>` part you can work with the app window and do a `appWindow.TitleBar.ExtendsContentIntoTitleBar = true;`

The first part is important. otherwise it won't work changing the colors etc.

**Reply by allens01** [reply]
Do you have a working example of this, where you have changed the color of the title bar?

**Reply by Yu-Core** [reply]
I searched for a lot of information, made some attempts, and finally got some results. I'm not sure if it's what you want

[https://github.com/Yu-Core/MauiWindowsTitleBarColor](https://github.com/Yu-Core/MauiWindowsTitleBarColor)

https://user-images.githubusercontent.com/96511239/235290220-2de573ad-1f37-46f7-ac84-56c804f333d7.mp4
